test/test.js

```diff
@@ -22,6 +22,14 @@ var MOCK_JS_VARIABLES =  {
 	'js-defined-no-prefix': '#ff0000'
 };
 
+var INVALID_JS_VARIABLES = {
```

Comment:
Perhaps rename to `NON_STRING_VARIABLES`

Reply:
Agree too. It is shonie@2c651e8f3df1b835aa023375d74a2306c9bfef96 too. 

Can i squash commits now? 

Thank you for review!

Reply:
Squash welcome 🙂

Reply:
Done!